database/migrations/2018_09_01_000001_create_permission_tables.php

```diff
@@ -9,79 +9,107 @@
 return new class() extends Migration {
     public function up(): void
     {
+        $teams = config('permission.teams');
```

Comment:
I'm a bid concerned about conditional migration code. Especially in files from migrations that should have occurred in 2018.  How does this logic impact the prod DB?

Reply:
that's from a vendor package update - only updated to keep things clean.
otherwise i absolutely agree, our application's own migrations should not be treated like this if possible.

Reply:
maybe we should actually incorporate the teams feature 🤔 i'll look into this.

Reply:
Reset this migration to its previous state and added a new migration for the teams feature as per the package's documentation: https://spatie.be/docs/laravel-permission/v6/basic-usage/teams-permissions#content-enabling-teams-permissions-feature